modules/external/controller/Falcon.ts

```diff
@@ -115,29 +115,6 @@ class Falcon {
   getLink(componentName: string, slug: string) {
     return `http://${componentName}-${slug}.kbox.k8s.xcaliber.io`;
   }
-
-  // TODO: mark properly the commit as success/fail like CircleCI does (needs to create a app for that)
-  resetAndAddTags(prNumber: number, toBeAdded: string) {
-    git.getLabels(prNumber, (data: any[]) => {
-      const pending = config.status.pending.tag;
-      const success = config.status.success.tag;
-      const fail = config.status.fail.tag;
-      data.map(item => {
-        if (item.name === success && toBeAdded !== success) {
-          git.removeLabel(prNumber, success);
-        }
-
-        if (item.name === fail && toBeAdded !== fail) {
-          git.removeLabel(prNumber, fail);
-        }
-
-        if (item.name === pending && toBeAdded !== pending) {
-          git.removeLabel(prNumber, pending);
-        }
-      });
-      git.addLabels(prNumber, [toBeAdded]);
-    });
-  }
```

Comment:
Was trying to simplify the flow and saw this being called. I didn't see what it had to do with Falcon management :P